spec/spec_helper.rb

```diff
@@ -105,6 +105,9 @@
       options[:re_record_interval] = nil if /bing.?v5/i === name
       options[:re_record_interval] = nil if /google|gss/i === name
 
+      # disable re-recording application_document_title cassettes which relies on tika server.
+      options[:re_record_interval] = nil if /application_document_title/ === name
```

Comment:
@MothOnMars could you weigh on whether you want this change?

Reply:
@nickmarden @viclim , the short answer is yes, but we should do this a different way. 

The backstory on re-recording cassettes is that we originally decided to re-record everything monthly in case there were any external API changes that we weren't aware of. That was later bumped to bi-monthly, and I recently turned off re-recording entirely for cucumber tests. At this point, I'm inclined to turn off re-recording for everything but BingV6-related specs, but I would like to do that in a separate commit (on its way).